controller/api/destination/profile_translator.go

```diff
@@ -78,9 +75,9 @@ func toDuration(d time.Duration) *duration.Duration {
 	}
 }
 
-// toServiceProfile returns a Proxy API DestinationProfile, given a
+// createDestinationProfile returns a Proxy API DestinationProfile, given a
```

Comment:
what's the motivation behind this rename?

Reply:
It felt misnamed as it takes a ServiceProfile and returns a DestinationProfile. I'd expect something named `toServiceProfile` to return a ServiceProfile.